messagebird/conversation_channel.py

```diff
@@ -0,0 +1,27 @@
+from messagebird.base import Base
+
+
+class Channel(Base):
+    def __init__(self):
+        self.id = None
+        self.name = None
+        self.platformId = None
```

Comment:
shouldn't we use snake case for variable names in python?

Reply:
yes, but it has to be like that because the Base class have a load from json and it match only if the key is the same as the json, if I update this is not going to work